WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -32,8 +45,131 @@ struct WooShippingCreateLabelsView: View {
+                                    .truncationMode(.tail)
+                            }
+                            .padding()
+                            Divider()
+                            GridRow(alignment: .top) {
+                                Text(Localization.BottomSheet.shipTo)
+                                VStack(alignment: .leading) {
+                                    Text("1 Infinite Loop") // TODO: 14044 - Show real "ship to" address (customer address)
+                                        .bold()
+                                    Text("Cupertino, CA 95014")
+                                    Text("USA")
+                                }
+                            }
+                            .padding()
+                        }
+                        .font(.subheadline)
+                        .roundedBorder(cornerRadius: Layout.cornerRadius, lineColor: Color(.separator), lineWidth: 0.5)
+                        Group {
+                            AdaptiveStack {
+                                Image(uiImage: .productIcon)
+                                    .frame(width: Layout.iconSize)
+                                Text(viewModel.items.itemsCountLabel)
+                                    .bold()
+                                Spacer()
+                                Text("$148.50") // TODO: 14044 - Show real item total
+                            }
+                            AdaptiveStack {
+                                Image(uiImage: .shippingIcon)
+                                    .frame(width: Layout.iconSize)
+                                Text("Flat rate shipping") // TODO: 14044 - Show real shipping name
+                                    .bold()
+                                Spacer()
+                                Text("$25.00") // TODO: 14044 - Show real shipping amount
+                            }
+                        }
+
+                        Divider()
+                            .padding(.trailing, -16)
+                        Text(Localization.BottomSheet.shipmentCosts)
+                            .footnoteStyle()
+                        Group {
+                            AdaptiveStack {
+                                Text(Localization.BottomSheet.subtotal)
+                                Spacer()
+                                Text("$0.00") // TODO: 13555 - Show real subtotal value
+                                    .if(true) { subtotal in // TODO: 14044 - Only show placeholder if subtotal is not available
+                                        subtotal.redacted(reason: .placeholder)
+                                    }
+                            }
+                            AdaptiveStack {
+                                Text(Localization.BottomSheet.total)
+                                    .bold()
+                                Spacer()
+                                Text("$0.00") // TODO: 13555 - Show real total value
```

Comment:
This will depend on the selected shipping rate (and any add-ons, e.g. signature required).